help_to_heat/frontdoor/eligibility.py

```diff
@@ -30,25 +28,34 @@ def calculate_eligibility(session_data):
     country = session_data.get("country")
     benefits = session_data.get("benefits")
     property_type = session_data.get("property_type")
+    own_property = session_data.get("own_property")
 
     # "Scenario 0"
     if property_type == "Park home":
         return ("GBIS",)
-
-    # Scenario 1
+    
+    # ECO4 and GBIS scenario 1 - home owner
     if country in country_council_tax_bands:
-        if council_tax_band in country_council_tax_bands[country]["eligible"]:
-            if epc_rating in ("E", "F", "G", "Not found"):
+        if own_property in ("Yes, I own my property and live in it",):
+            if epc_rating in ("D","E", "F", "G", "Not found"):
                 if benefits in ("Yes",):
-                    logger.error("Scenario 1")
+                    logger.error("ECO4 scenario 1")
```

Comment:
Did we want to remove this logging?

Reply:
Happy either way. They should be released up through the envs together, so we definitely want to do it before releasing the changes to UAT.

Do you have thoughts on whether we should remove them or switch them to log at debug level? I'd lean towards remove (since they can be re-added as required if seeing scenario selection is useful), but you've been more hands-on with dev than me.

Reply:
I think they can be removed, I did quite a lot of manual testing and the logging proved useful, but I'm happy that it's served its purpose and can be removed